package-system/AWSGameLiftServerSDK/README.md

```diff
@@ -3,115 +3,30 @@
 You can find the official GameLift documentation [here](https://aws.amazon.com/documentation/gamelift/).
 
 ## Minimum requirements:
-* Either of the following:  
-  * Microsoft Visual Studio 2017 - Windows Build
-  * Clang 6.0 - Linux Build
-* CMake version 3.1 or later
-* Python version 3.6 or later
+* Either of the following:
+  * Microsoft Visual Studio 2019 - Windows Build
+  * Clang 12.0 and GCC 9.3 - Linux Build
+* CMake version 3.20 or later
+* Python version 3.7 or later
 * A Git client available on the PATH
 
-## Build the GameLiftServerSdk - Windows
-1. Download the latest [GameLift Server SDK](https://aws.amazon.com/gamelift/getting-started/) 
-   and unzip source `GameLift-Cpp-ServerSDK-{version-number}`
-   
-2. Copy and put `package-system/AWSGameLiftServerSDK/build_and_package.py` and `package-system/AWSGameLiftServerSDK/gamelift-sdk.json` 
-   into `GameLift-Cpp-ServerSDK-{version-number}` directory in step 1
-   
-3. Run the python script `build_and_package.py` under `GameLift-Cpp-ServerSDK-{version-number}` directory, like
-   ```
-   & python build_and_package.py --config gamelift-sdk.json --platform windows --version 3.4.1
-   ```
-   
-4. Build package will be located under `GameLift-Cpp-ServerSDK-{version-number}/build/{version-number}` directory
-
-5. Copy and put build package into `package-system/AWSGameLiftServerSDK/windows/AWSGameLiftServerSDK`, for example
-   ``` 
-   - AWSGameLiftServerSDK
-     - bin
-     - include
-     - lib
-     - LICENSE_AMAZON_GAMELIFT_SDK.txt
-     - NOTICE_C++_AMAZON_GAMELIFT_SDK.txt
-   ```
-6. Update `package-system/AWSGameLiftServerSDK/windows/PackageInfo.json` with built version `PackageName`
-
-7. Update `<3p-package-source path>/package_build_list_host_windows.json` with updated `PackageName` in step 6
+## Prepare steps:
+1. As there is no public repository for server sdk source code, please double check [Amazon GameLift Release Notes](https://docs.aws.amazon.com/gamelift/latest/developerguide/release-notes.html)
+   to confirm server sdk version information
+2. Modify `PACKAGE_VERSION`, `GAMELIFT_SERVER_SDK_RELEASE_VERSION` and `GAMELIFT_SERVER_SDK_DOWNLOAD_URL` to update version
 
 ## Build the GameLiftServerSdk - Linux
```

Comment:
This section only talks about how to link Clang compiler for cmake build but doesn't provide instructions to actually "build" the SDK. May worth rename the title here.

Reply:
will update

Reply:
Do you plan to add instructions for how to "build" here later? Or is the idea that this will now happen automatically when building the server target?

Reply:
There is no specific build instruction, as it follows the general workflow of 3p system